src/Engine/Texture/TextureAsset.cpp

```diff
@@ -32,18 +32,18 @@ void TextureAsset::Load(const std::string& name) {
     this->name = name.substr(pos + 1);
     path = name.substr(0, pos + 1);
     std::string filename = Hymn().GetPath() + "/" + name;
-    
+
     // Get properties from meta file.
     Json::Value root;
-    if(!FileSystem::FileExists(std::string(filename + ".json").c_str())) {
+    if (!FileSystem::FileExists(std::string(filename + ".json").c_str())) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') keyword 'if' not followed by a single space [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT008)